<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <node name="pick_objects" pkg="pick_objects" type="pick_objects" output="screen">
    <param name="drop_x" value="7.5"/>
    <param name="drop_y" value="1.0"/>
    <param name="drop_a" value="1.0"/>
    
    <param name="pick_x" value="-5.4"/>
    <param name="pick_y" value="6.0"/>
    <param name="pick_a" value="1.0"/>
  </node>
</launch>